### PR TITLE
Add double dash to confluent local produce/consume

### PR DESCRIPTION
**Pull request by ybyzek**

### CODE REVIEW

File: clients/cloud/confluent-cli/README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ $ ccloud topic create test1
 * `--property parse.key=true --property key.separator=,`: pass key and value, separated by a comma
 
 ```bash
-$ confluent local produce test1 --cloud --property parse.key=true --property key.separator=,
+$ confluent local produce -- test1 -- --cloud --property parse.key=true --property key.separator=,
```
Comment (ghost) [inline clients/cloud/confluent-cli/README.md]: Is a double-dash needed on both sides of `test1` here? It doesn't look that way below for `confluent local consume`

File: clients/cloud/confluent-cli/README.md [hunk 2]
```diff
@@ -108,7 +108,7 @@ $ ccloud topic create test2
 * `--property schema.registry.basic.auth.user.info`: <SR API KEY>:<SR API SECRET> 
 
 ```bash
-$ confluent local produce test2 --cloud --value-format avro --property value.schema='{"type":"record","name":"myrecord","fields":[{"name":"count","type":"int"}]}' --property schema.registry.url=https://<SR ENDPOINT> --property basic.auth.credentials.source=USER_INFO --property schema.registry.basic.auth.user.info='<SR API KEY>:<SR API SECRET>'
+$ confluent local produce -- test2 --cloud --value-format avro --property value.schema='{"type":"record","name":"myrecord","fields":[{"name":"count","type":"int"}]}' --property schema.registry.url=https://<SR ENDPOINT> --property basic.auth.credentials.source=USER_INFO --property schema.registry.basic.auth.user.info='<SR API KEY>:<SR API SECRET>'
```
Comment (ghost) [inline clients/cloud/confluent-cli/README.md]: Is .a double-dash needed _before_ `test2` here?

File: clients/cloud/confluent-cli/README.md (hunk 1 above)
Comment (ybyzek) [reply clients/cloud/confluent-cli/README.md]: Nope, great catch!

File: clients/cloud/confluent-cli/README.md (hunk 2 above)
Comment (ybyzek) [reply clients/cloud/confluent-cli/README.md]: Nope, great catch again

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**